packages/crowdstrike/data_stream/fdr/_dev/deploy/tf/main.tf

```
@@ -0,0 +1,87 @@
+variable "TEST_RUN_ID" {
+  default = "detached"
+}
+
+provider "aws" {
+  default_tags {
+    tags = {
+      environment  = var.ENVIRONMENT
+      repo         = var.REPO
+      branch       = var.BRANCH
+      build        = var.BUILD_ID
+      created_date = var.CREATED_DATE
+    }
+  }
+}
+
+output "queue_url" {
+  value = aws_sqs_queue.crowdstrike_queue.url
+}
+
+# Common queue
+resource "aws_sqs_queue" "crowdstrike_queue" {
+  name       = "elastic-package-crowdstrike-queue-${var.TEST_RUN_ID}"
+  policy     = <<POLICY
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "sqs:SendMessage",
+      "Resource": "arn:aws:sqs:*:*:elastic-package-crowdstrike-data-queue-${var.TEST_RUN_ID}",
```

The resource here should be the queue. This is a policy for the queue that allows the two buckets to send messages to it. So the name here must match the queue name.

```suggestion
      "Resource": "arn:aws:sqs:*:*:elastic-package-crowdstrike-queue-${var.TEST_RUN_ID}",
```